utils/src/ooinstall/cli_installer.py

```diff
@@ -516,6 +516,13 @@ def error_if_missing_info(oo_cfg):
     if missing_info:
         sys.exit(1)
 
+def get_proxy_hostname_and_excludes():
+    message = "Specify the hostname for your proxy? (ENTER for none)"
```

Comment:
I think we should put a little more information here about why they might need a proxy.  Maybe something like this:

```
If a proxy is needed to reach HTTP and HTTPS traffic please enter the name below.  This proxy will be configured by default for all processes needing to reach systems outside the cluster.  More advanced configuration is possible if using ansible directly:

<insert link to official docs>
```